### ISSUE TITLE
How to display a custom text when hovered over a cell 

### ISSUE THREAD

**Issue by narendravenkata** [react-bootstrap-table/react-bootstrap-table2#3]
The table currently has "headerText" which we can use to display custom text on the Table Header but for the cell I didnt find anything like this to use

**Comment by AllenFang** [comment]
yes, it's a issue we dont support on `react-bootstrap-table`, I will support it on 1 and 2, thanks your feedback 👍
